src/index.html

```diff
@@ -7,12 +7,54 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="/src/styles/index.scss"
```

Comment:
Link to the SCSS file is incorrect. Since you are using SCSS, which needs to be compiled to CSS, the `href` should point to the compiled CSS file, not the SCSS file directly. Make sure the path is correct and points to the CSS file that the browser can understand.